pages/careers/_open-positions.html

```diff
@@ -88,22 +88,108 @@ <h4>Your Ideal Background</h4>
 
     <hr />
 
-    <h3>Senior Software Engineer</h3>
+    <a target="site-reliability-engineer-sre"></a>
+    <h3>Site Reliability Engineer (SRE)</h3>
+    <h4>
+      Be an SRE without being on call! Help Gruntwork develop SRE as a product
+      offering and enable our hundreds of customers to achieve best-practices
+      for operations and reliability, all while not being woken up at 3am!
+    </h4>
```

Comment:
```suggestion
    <p>
      Be an SRE without being on call! Help Gruntwork develop SRE as a product
      offering and enable our hundreds of customers to achieve best-practices
      for operations and reliability, all while not being woken up at 3am!
    </p>
```

It's cleaner to have this information hierarchy:
![image](https://user-images.githubusercontent.com/4295964/84607138-6ef70f80-ae60-11ea-98a8-0ebf5331c329.png)

versus the original one:
![image](https://user-images.githubusercontent.com/4295964/84607145-7b7b6800-ae60-11ea-8d52-359e4df0e117.png)

Reply:
2329f15c88a9a51c5fd2a6de4fdce92ffe02d8d2